docs/ops/deployment/kubernetes.md

```diff
@@ -159,6 +159,8 @@ with the `kubectl` command:
 ## Appendix
 
 ### Common cluster resource definitions
+The Deployment definitions use the pre-built image `flink:{% if site.is_stable %}{{site.version}}-scala{{site.scala_version_suffix}}`{% else %}latest` (which contains the latest released Flink built against Scala 2.12){% endif %} which can be found [on Docker Hub](https://hub.docker.com/_/flink/).
+The image is built from this [Github repository](https://github.com/apache/flink-docker).
```

Comment:
We already have a chapter on this page about the image, I would rather refer to it: `#flink-docker-image`.
This way we would have to change the image description only in one place. Also, the information about scala 2.12 for latest can become stale, it would make sense to just refer to the tags page. We have everywhere Scala 2.11 by default except docker and might want to make it 2.11 for docker as well later.

Reply:
Yes, I fixed this. I've added a bit in the central docker documentation files instead.